Code/Tools/AssetProcessor/native/AssetManager/SourceFileRelocator.cpp

```diff
@@ -167,19 +167,27 @@ Please note that only those seed files will get updated that are active for your
         SourceFileRelocationContainer& sources) const
     {
         QHash<QString, int> sourceIndexMap;
+        QSet<QString> filesNotInAssetDatabase;
         for (auto& file : pathMatches)
         {
             QString databaseSourceName;
 
             PlatformConfiguration::ConvertToRelativePath(file, scanFolderInfo, databaseSourceName);
-            m_stateData->QuerySourceBySourceNameScanFolderID(databaseSourceName.toUtf8().constData(), scanFolderInfo->ScanFolderID(), [this, &sources, &scanFolderInfo, &sourceIndexMap, &databaseSourceName](const AzToolsFramework::AssetDatabase::SourceDatabaseEntry& entry)
+            filesNotInAssetDatabase.insert(databaseSourceName);
+            m_stateData->QuerySourceBySourceNameScanFolderID(databaseSourceName.toUtf8().constData(), scanFolderInfo->ScanFolderID(), [this, &sources, &scanFolderInfo, &sourceIndexMap, &databaseSourceName, &filesNotInAssetDatabase](const AzToolsFramework::AssetDatabase::SourceDatabaseEntry& entry)
                 {
                     sources.emplace_back(entry, GetProductMapForSource(entry.m_sourceID), entry.m_sourceName, scanFolderInfo);
+                    filesNotInAssetDatabase.remove(databaseSourceName);
                     sourceIndexMap[databaseSourceName] = aznumeric_cast<int> (sources.size() - 1);
                     return true;
                 });
         }
 
+        for (const QString& file : filesNotInAssetDatabase)
+        {
+            AZ_Printf("AssetProcessor", "File `%s` was found/matched but is not a source asset.  Skipping.\n", file.toUtf8().constData());
```

Comment:
Am slightly confused by output, surely file is a "source asset" ie its an asset on disc, its just that we do not have a mapping in the asset database (ie it failed to process?)

Should this be more friendly to user?
```suggestion
            AZ_Printf("AssetProcessor", "File `%s` matched during relocation does not have entry in Asset database.  Skipping.\n", file.toUtf8().constData());
```

Do we want to add any information about how to fix? ie

```
if  (!filesNotInassetDatabase.empty(){
    AZ_Printf("AssetProcessor", "The following files were found without entries in the asset database. Examine AP logs to identify reasons for why asset was ignored or failed to process. "
}
```

Reply:
I am not sure users conceive a source asset as an entry in the asset database, or a file on disk. Your suggestion is probably better, as it clearly points out the problem is "no entry in the database".
However, I feel a bit confused by a file being "matched" but "does not have entry in the Asset DB". Should we keep the wording "found" such that it is clear that the file "exists on disk"?
e.g. does this sound better to you? "File `%s` founded on disk but does not have an entry in Asset database.  Skipping this relocation.\n"

Reply:
Current output example:
```
AssetProcessor: File `Assets/Test10641/test - Copy (3).mtl` was found/matched but is not a source asset.  Skipping.
AssetProcessor: File `Assets/Test10641/test - Copy (2).mtl` was found/matched but is not a source asset.  Skipping.
AssetProcessor: File `Assets/Test10641/test.mtl` was found/matched but is not a source asset.  Skipping.
AssetProcessor: File `Assets/Test10641/test - Copy.mtl` was found/matched but is not a source asset.  Skipping.
AssetProcessor: Wildcard search matched one or more files but none are source assets.  This utility only handles source assets.
```